Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml

```
@@ -937,9 +937,26 @@
 - type: entity
   name: bread dog
   id: MobBreadDog
-  parent: FoodBreadSausage
+  parent: EdibleBase
   description: It's a bread. It's a dog. It's a... breaddog?
   components:
+  - type: FlavorProfile
+    flavors:
+      - meaty
+      - bread
```

```suggestion
    flavors:
    - meaty
    - bread
```
minor yaml convention nitpick